app/src/main/java/com/pajato/android/gamechat/exp/ChessFragment.java

```diff
@@ -446,147 +350,162 @@ private void updateExperience() {
         setPlayerName(R.id.player2Name, 1, model);
         setPlayerWinCount(R.id.player1WinCount, 0, model);
         setPlayerWinCount(R.id.player2WinCount, 1, model);
-//        setPlayerSymbol(R.id.player1Symbol, 0, model);
-//        setPlayerSymbol(R.id.player2Symbol, 1, model);
         setPlayerIcons(model.turn);
         setGameBoard(model);
         setState(model);
     }
 
+    /**
```

Comment:
This method has been improved is my guess.  While that might be true, it is still too big, IMHO.  No reason to shrink it right now though.